File: app/src/main/java/com/example/pam_project/comparators/ListAlphabeticalComparator.java
package com.example.pam_project.comparators;

import com.example.pam_project.features.lists.list.ListInformation;

public class ListAlphabeticalComparator extends ListInformationComparator {

    @Override
    public int compare(ListInformation o1, ListInformation o2) {
        return String.CASE_INSENSITIVE_ORDER.compare(o1.getTitle(), o2.getTitle());
    }
}

File: app/src/main/java/com/example/pam_project/comparators/ListAlphabeticalComparator.kt
package com.example.pam_project.comparators

import com.example.pam_project.features.lists.list.ListInformation

class ListAlphabeticalComparator : ListInformationComparator() {
    override fun compare(o1: ListInformation?, o2: ListInformation?): Int {
        return String.CASE_INSENSITIVE_ORDER.compare(o1?.title, o2?.title)
    }
}